Neos.Flow/Classes/ObjectManagement/CompileTimeObjectManager.php

```diff
@@ -237,12 +237,6 @@ protected function registerClassFiles(array $packages): array
                 }
                 if ($package instanceof FlowPackageInterface && $shouldRegisterFunctionalTestClasses) {
                     foreach ($package->getFunctionalTestsClassFiles() as $fullClassName => $path) {
-                        if (PHP_VERSION_ID <= 80000 && str_contains($fullClassName, '\\PHP8\\')) {
```

Comment:
do we want to comment this out instead for preparation of PHP83 fixtures 😅

Reply:
Nah.

Reply:
Never leave dead code, in my opinion.